ipn/ipnstate/store.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ipnstate
```

Comment:
// Package ipnstate ....

(The primary file in a package should have a package comment)

Reply:
I believed this was covered by https://github.com/tailscale/tailscale/blob/78d4c561b582c97bfe9e4c9f41afbe0cd545d8b3/ipn/ipnstate/ipnstate.go#L5-L8

:thinking:

Reply:
oh!! We already _had_ an `ipnstate` package. Right. I was proposing adding a new one.

But I don't think we can merge this functionality into the existing `ipnstate` package for the reason it says: it's intended to be a leaf dep-free/minimal package.

Reply:
no worries, that did not feel right indeed :grimacing: do you have another naming suggestion?

Reply:
what do you think of using `ipnstore` as we did initially and move the `StateStore` nomenclature into something like `Provider`? :thinking: